include/zim/archive.h

```diff
@@ -220,8 +220,12 @@ namespace zim
 
       /** Get an entry using a path.
        *
-       *  Get an entry using its path.
-       *  The path must contains the namespace.
+       *  Search an entry in the zim, using its path.
+       *  On archive with new namespace scheme, path must not contain the namespace.
+       *  On archive without new namespace scheme, path must contain the namespace.
+       *  A compatibility layer exists to accept "old" path on new archive (and the opposite)
+       *  to help using saved path (bookmark) on new archive.
+       *  But it is not possible to access content in other namespace than `C` in new archive.
```

Comment:
This description doesn't document the potential problems with the compatibility layer:

> This can lead to false results. Suppose that the ZIM file (utilizing the new namespace scheme) doesn't contain entries under `Q/` or `C/Q/` path prefix. Yet the request for `Q/something` will be converted to `C/something` and that item will be returned if it exists.

Overall, my impression is that this point in the description of #843 didn't receive the attention it deserves.